index.html

```diff
@@ -153,6 +153,21 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <p>
+          To <dfn data-export>create a CropTarget</dfn> with <var>element</var> as input, run the following steps:
+          <ol>
+            <li>
+              <p>Let <var>cropTarget</var> be a new object of type {{CropTarget}}.</p>
+            </li>
+            <li>
+              <p>Create <var>cropTarget</var>.<dfn data-dfn-for="CropTarget">[[\Element]]</dfn> initialized to <var>element</var>.</p>
+              <div class="note">
+                <p><var>cropTarget</var> is keeping a weak reference to the element it is representing.
+                  In other words, <var>cropTarget</var> will not prevent garbage collection of its element.</p>
```

Comment:
```suggestion
                The user agent must keep a weak association between this <var>cropTarget</var> and the element
                it is representing. In other words, <var>cropTarget</var> will not prevent garbage collection of its
                element.</p>
```

Reply:
@youennf does this lgty?

Reply:
The aim of this paragraph as I read it is to ensure cropTarget _"will not prevent garbage collection of its element"_, which needs to be said here to not imply otherwise.

The [common](https://html.spec.whatwg.org/#groupings-of-browsing-contexts) pattern is to say "weak reference", or "weak map" to describe these garbage collection requirements.

I've s/weak association/weak [=map=]/

Reply:
For serialisation, we can simply store the weak reference as a serialisation slot. This works great. I do not see the issue on the contrary, it makes things much simpler.